packages/lodestar-cli/src/commands/dev/command.ts

```diff
@@ -88,12 +89,14 @@ export class DevCommand implements ICliCommand {
     }
     resetPath(conf.db.name);
 
-    conf.network.discv5 = Object.assign(
-      {},
-      {enr: ENR.createFromPeerId(peerId), bindAddr: "/ip4/0.0.0.0/udp/5501"},
-      conf.network.discv5,
-    );
-    const libp2p = await createNodeJsLibp2p(peerId, conf.network);
+    const defaultDiscv5Opt = {
+      enr: ENR.createFromPeerId(peerId),
+      bindAddr: "/ip4/0.0.0.0/udp/5501",
```

Comment:
```suggestion
      bindAddr: "/ip4/127.0.0.1/udp/0",
```
should bind on random port and so commands in readme still works.